Actions/CheckForUpdates/CheckForUpdates.ps1

```diff
@@ -220,16 +220,26 @@ try {
                         }
                     }
 
-                    # The PullRequestHandler workflow can have a RepoSetting called CICDPullRequestBranches, which will be used to set the branches for the workflow
                     if ($baseName -eq "PullRequestHandler") {
+                        # The PullRequestHandler workflow can have a RepoSetting called PRBuildTrigger which specifies the trigger to use for Pull Requests
+                        $triggerSection = $yaml.Get('on:/pull')
+                        $prTrigger = "pull_request_target" # Default trigger: https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target
```

Comment:
The typical way to add settings with a default is to add it in AL-Go-Helper:
https://github.com/microsoft/AL-Go/blob/78365a57f441dbe471d90aa60a3696c7fa2c1152/Actions/AL-Go-Helper.ps1#L524

with the default.
Then settings will be applied from all settings files and type checked - and you can safely use reposettings.setting here,

Reply:
I'll adjust :)